ocs_ci/framework/main.py

```diff
@@ -66,11 +69,42 @@ def init_ocsci_conf(arguments=None):
             os.path.expanduser(framework.config.RUN['bin_dir'])
         )
         utils.add_path_to_env_path(framework.config.RUN['bin_dir'])
+    print(framework.config)
     check_config_requirements()
 
 
+def init_version_defaults(arguments=None):
+    """
+    Update the config object with version specific defaults
+
+    Args:
+        arguments (list): Arguments for pytest execution
+
+    """
+    print(arguments)
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument(
+        '--ocs-version',
+        default=framework.config.ENV_DATA['current_default_ocs'],
+        choices=framework.config.ENV_DATA['available_ocs_versions']
```

Comment:
I think we could get rid of this, because we might have more choice and it will require back and forth of PRs, maybe later on add some other kind of verification to ensure its accurate, ?

Reply:
I don't see how we can get around PRs to add choices to this list if we decide to do any form of verification. I think we have to decide whether we want a legitimate list of options (that will require a PR to update) or to let the user pass whatever value they want.